pkg/ubinary/little_endian.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build 386 amd64 arm arm64 mipsle mips64le ppc64le
+// +build 386 amd64 arm arm64 mipsle mips64le ppc64le riscv riscv64
```

Comment:
oh dear. the naming is not standard, is it? Shouldn't it be rv32 and rv64?

Reply:
These values have already been reserved in go1.11. See: https://golang.org/doc/go1.11#riscv